practice-two/src/styles/components/all.css

```diff
@@ -1,5 +1,5 @@
 @import url("button.css");
-@import url("input.css");
+@import url("form.css");
 @import url("typography.css");
 @import url("image.css");
 @import url("card.css");
```

Comment:
As we discussed before, we just use single quote or double quote for consistent, right? Which one you choose?

Reply:
I will choose double quotes for all of css files, and I will fix the base/typography.css as it uses single quote.

Reply:
Got you. Pls re-open it so I can merge